### PR TITLE
Renaming

### PR DESCRIPTION
**Pull request by lightsprint09**
Big renaming to fulfill the 1.0 release
Fixes: #62 

#### Make sure to check all boxes before merging

- [x] Method/Class documentation
- [x] README.md documentation
- [x] Unit tests for new features/regressions

### CODE REVIEW

File: Source/BasicNetworkService.swift [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+//  THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+//  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+//  DEALINGS IN THE SOFTWARE.
+//
+//  Created by Lukas Schmidt on 31.08.16.
```
Comment (IsabellSchmelzeisen) [inline Source/BasicNetworkService.swift]: Wrong Header

File: README.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ networkService.request(resource, onCompletion: { htmlText in
 
 ```
 
-## Loade types conforming to `Decodable`
+## Loade types conforming to Swift-`Decodable`
```
Comment (IsabellSchmelzeisen) [inline README.md]: Load

File: Source/NetworkService.swift [hunk 3]
```diff
@@ -22,39 +22,71 @@
 //  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 //  DEALINGS IN THE SOFTWARE.
 //
-//  Created by Lukas Schmidt on 31.08.16.
+//  Created by Lukas Schmidt on 21.07.16.
```
Comment (IsabellSchmelzeisen) [inline Source/NetworkService.swift]: Wrong Header

File: Source/BasicNetworkService.swift (hunk 1 above)
Comment (lightsprint09) [reply Source/BasicNetworkService.swift]: 😂😡

Comment (lightsprint09) [reply Source/BasicNetworkService.swift]: I tried to fix all headers in #56

**Pull request merged**